components/Button.jsx

```diff
@@ -1,7 +1,13 @@
 import Link from 'next/link';
 
 const Button = ({ href, children, size = 'small' }) => {
-  return <p><Link href={href}><a className={`usa-button usa-button--${size}`} href={href}>{children}</a></Link></p>
+  console.log(href);
+  return <p>
+    <Link href="/[pid]" as={href}>
```

Comment:
Link needs to take

`href` as the actual file we are routing to ([pid].js)
`as` as the URL we show in the users browser (/request)